src/Layer/GeometryLayer.js

```diff
@@ -0,0 +1,178 @@
+                        return;
+                    }
+                    this.changeOpacity(object);
+                    // 3dtiles layers store scenes in children's content property
+                    if (object.content) {
+                        object.content.traverse(this.changeOpacity);
+                    }
+                });
+            }
+        });
+
+        this.attachedLayers = [];
+        this.visible = true;
+
+        // Attached layers expect to receive the visual representation of a
+        // layer (= THREE object with a material).  So if a layer's update
+        // function don't process this kind of object, the layer must provide a
+        // getObjectToUpdateForAttachedLayers function that returns the correct
+        // object to update for attached layer.
+        // See 3dtilesProvider or PointCloudProvider for examples.
+        // eslint-disable-next-line arrow-body-style
+        this.getObjectToUpdateForAttachedLayers = (obj) => {
+            if (obj.parent && obj.material) {
+                return {
+                    element: obj,
+                    parent: obj.parent,
+                };
+            }
+        };
+
+        this.postUpdate = () => {};
+    }
+
+    /**
+     * Attach another layer to this one. Layers attached to a GeometryLayer will
+     * be available in <code>geometryLayer.attachedLayers</code>. The layer will
+     * also have a boolean property <code>isAttached</code> set to
+     * <code>true</code>.
+     *
+     * @param {Layer} layer - The layer to attach, that must have an
+     * <code>update</code> method.
+     */
+    attach(layer) {
+        if (!layer.update) {
+            throw new Error(`Missing 'update' function -> can't attach layer
+                ${layer.id}`);
+        }
+        this.attachedLayers.push(layer);
+        layer.isAttached = true;
+    }
+
+    /**
+     * Detach a layer attached to this one. See {@link attach} to learn how to
+     * attach a layer.
+     *
+     * @param {Layer} layer - The layer to detach.
+     *
+     * @return {boolean} Confirmation of the detachment of the layer.
+     */
+    detach(layer) {
+        const count = this.attachedLayers.length;
+        this.attachedLayers = this.attachedLayers.filter(attached => attached.id != layer.id);
+        layer.isAttached = false;
+        return this.attachedLayers.length < count;
+    }
+
+    /**
+     * Picking method for this layer. It uses the {@link Picking#pickObjectsAt}
+     * method.
+     *
+     * @param {View} view - The view instance.
+     * @param {Object} coordinates - The coordinates to pick in the view. It
+     * should have at least <code>x</code> and <code>y</code> properties.
+     * @param {number} radius - Radius of the picking circle.
+     *
+     * @return {Array} An array containing all targets picked under the
+     * specified coordinates.
+     */
+    pickObjectsAt(view, coordinates, radius) {
```

Comment:
why radius parameter don't use default value?
`pickObjectsAt(view, coordinates, radius = this.config.defaultRadius)`

Reply:
Thanks, didn't think of it this way !